src/views/components/logout.js

```diff
@@ -0,0 +1,27 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import React, {useState} from 'react';
+import {View, Text,StyleSheet,Button} from 'react-native';
+import {logOut} from '../../services/AsyncStorageService'
+
+export default function Logout({navigation}) {
+  const logout = async () => {
+    logOut
```

Comment:
what it **logOut** here?

Reply:
it's a screen when the user clicks the logout button whole data will clear and navigate the login screen
.